algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -448,6 +450,50 @@ case class CMSItem[K: Ordering](item: K, override val params: CMSParams[K]) exte
 }
 
 /**
+ * Used for holding a small set of elements.
+ */
+case class CMSItems[K: Ordering](items: List[K], override val totalCount: Long, override val params: CMSParams[K]) extends CMS[K](params) {
+
+  override def +(x: K, count: Long): CMS[K] = CMSItems(x :: items, totalCount + 1, params).toCMS
```

Comment:
It looks as if `count` is not being taken into account.  Shouldn't `+(x, 4)` result in the count of `x` being increased by 4?

This was the reason why I asked the `totalCount` question about `items.size` vs. `items.sum` (the L1-norm) -- size would be equivalent to sum only if we restrict `+` to `+(x, 1)`.